src/components/table/column-picker/column-picker.component.ts

```diff
@@ -316,7 +328,7 @@ export class ColumnPickerComponent implements OnChanges {
     }
 
     /** Get the column name based on type */
-    private getColumnName(item: string | ColumnPickerGroupItem): string {
+    getColumnName(item: string | ColumnPickerGroupItem): string {
```

Comment:
We usually prefix private methods that need to be called in the view with an underscore.